### ISSUE TITLE
Move the commands to bash scripts

### ISSUE THREAD

**Issue by cmoulliard** [bucharest-gold/litoria#3]
The .travis.yml file contains npm, git & shell commands developed using Travis-ci syntax. Sych commands should be move to bash scripts packaged under build directory

**Comment by helio-frota** [comment]
bucharest-gold/entente#133

**Comment by helio-frota** [comment]
Makes sense to close this and create a new one to move to github actions ?
